plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -6,15 +6,37 @@ import { Button, Tooltip } from '@wordpress/components';
 import { getNewPath } from '@woocommerce/navigation';
 import { help } from '@wordpress/icons';
 import { Table } from '@woocommerce/components';
+import { useContext, useEffect, useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { getAdminSetting } from '~/utils/admin-settings';
+import { getAdminSetting } from '../../../utils/admin-settings';
```

Comment:
I am also not a fan of the `~` sign :smile:

Reply:
It was throwing a warning, I would ignore it otherwise.